homework/index.js

```diff
@@ -0,0 +1,64 @@
+function getPromise() {
+  const ret = fetch('https://api.github.com/orgs/HackYourFuture/repos?per_page=100')
+    .then(response => response.json());
+  return ret;
+}
+
+
+function populate(Prom) {
+  const selectContent = document.getElementById('liste').value;
+  Prom.then((result) => {
+    result.forEach((x) => {
+      if (x.name === selectContent) {
+        document.getElementById('show').innerHTML = `
+         <b>Repository :</b>   <a href="">${x.name} </a><br>
+          <b>Descripion :</b>   ${x.description} <br>
+          <b>Forks :</b>   ${x.forks} <br>
+          <b>Updated :</b>   ${new Date(x.updated_at).toLocaleString('en-US')} <br>`;
+      }
+    });
+  });
+}
+
+
+function loadPage(Prom) {
+  Prom.then((result) => {
+    const nameArray = [];
+    // make an array of all the names
```

Comment:
Short comments on same line they describe; or if it's too long, on the line *above*. Also, generally people start comments with a capital (but rarely add a full stop at the end; unless it's multiple sentences). (nit-pick again ;) )

More importantly, though, instead of first declaring this array and then populating it; this would be an ideal candidate where to use `map()` I think! You can then easily do this in one line, instead of 5 - and it would be so clear that it doesn't need the comment! ;)